syncback/Tiltfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: Python -*-
+
+krsync_path = os.path.join(os.getcwd(), 'krsync.sh')
+
+
+def syncback(name, deploy, paths, target_dir='.'):
+    # name (str) = resource name
+    # k8s_identifier (str) = thing to exec on (pod, deployment/blah, etc)
```

Comment:
hmmmm....i'm confused...certain parts of this extension say that k8s_identifier can be a pod, but in krsync.sh it says it has to be a deployment?

Reply:
whoops, this comment is out of date. yes it has to be a deployment. (how important do you think it is that people be able to pass a different identifier--e.g. a pod, job name, etc.?)

Reply:
ya, jobs and statefulsets are pretty popular. is it hard to require the resource name? like "deployment/my-deployment"

fwiw, i found the code that kubectl uses to find the pod for an exec recently! https://github.com/kubernetes/kubectl/blob/master/pkg/polymorphichelpers/helpers.go#L84